.bazelrc

```diff
@@ -0,0 +1 @@
+build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
```

Comment:
What does this do? Default to building Linux binaries? Could you add a comment?

Reply:
I didn't mean to add it. I cross compile from my mac with M1 silicon but I don't think this should be the default.